src/main/java/doit/jpastudy2/repository/Attendant.java

```diff
@@ -0,0 +1,41 @@
+package doit.jpastudy2.repository;
+
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Attendant {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+
+    @Column(name = "n주차")
+    private int id;
```

Comment:
우선 자동으로 할당해주신 점 좋습니다!
n주차라고 작성해주신 것 보니, 아마 1주차에 해당하는 출석들을 모아서 설정하시려는 것 같은데 맞을까요?
맞다면 요 로직보다는 다음과 같이 작성해주시는 편이 좋을 것 같습니다

```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "attendant_id")
    private int id;

    @Column(name = "attendant_week_number")
    private int weekNumber;
```
pk에는 의미가 없는 인공적인 값을 넣어주시는 것이 DB 구조 상 로직 작성하기에 편하실 것 같습니다! 참고하시면 될 것 같아요